packages/components/RELEASE.md

```diff
@@ -4,13 +4,40 @@ Whenever there is an update to the components, and these changes have been appro
 
 Please see the instructions in the [CONTRIBUTING](CONTRIBUTING.md) file for more details about how to setup the project and make changes to the code for these packages.
 
-## Bump and release
+## The release process
 
-Follow the instructions for Changesets in the root [README](../../README.md).
+Release PRs, titled 'Version Packages', are created and/or automatically updated on every PR merge by the [changeset GitHub action](https://github.com/changesets/action).
 
-🚨 **DON'T FORGET**:
+In preparation for a release, login to Vercel CLI and create a canonical URL for the upcoming version:
 
-You need to communicate to the product teams that are consuming the components!
+1. Type `vercel login` in your terminal and select 'Continue with SAML Single Sign-On'
```

Comment:
If I type `vercel login` in my terminal, it says `command not found: vercel`. I imagine we need to install something, can you add a note about what/how should be installed?

Reply:
I've added an initial step for the Vercel CLI, pointing to their docs (as people may use npm,pnpm, or yarn to manage their global packages.